.github/workflows/autofix.yml

```diff
@@ -0,0 +1,45 @@
+---
+# The workflow name **must** be "autofix.ci" for Autofix CI to function correctly.
+# Any deviation from this name will cause Autofix CI to fail, as it relies on this 
+# specific identifier for execution. This is a strict requirement of Autofix CI.
+name: autofix.ci
+on: pull_request
+permissions: {}
+jobs:
+  autofix:
+    runs-on: ubuntu-24.04
+    permissions: {}
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - uses: aquaproj/aqua-installer@e2d0136abcf70b7a2f6f505720640750557c4b33 # v3.1.1
+        with:
+          aqua_version: v2.43.0
+
+      - name: Update aqua-checksums.json
+        run: aqua upc -prune
+
+      # go mod tidy
+      - uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # v5.3.0
+        with:
+          go-version-file: go.mod
+      - run: go mod tidy
+
+      # gofumpt
+      - name: Get changed Go files
+        id: changed-files
+        uses: tj-actions/changed-files@d6e91a2266cdb9d62096cebf1e8546899c6aa18f # v45.0.6
+        with:
+          use_rest_api: "true"
+          files: |
+            **/*.go
+      - run: |
+          # shellcheck disable=SC2086
+          gofumpt -l -w $ALL_CHANGED_FILES
+        if: steps.changed-files.outputs.all_changed_files_count != '0'
+        env:
+          ALL_CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
```

Comment:
```suggestion
      - if: steps.changed-files.outputs.all_changed_files_count != '0'
        env:
          ALL_CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
        run: |
          # shellcheck disable=SC2086
          gofumpt -l -w $ALL_CHANGED_FILES
        
```

Reply:
Just to confirm. You only changed the order, right?